markets/perps-market/contracts/modules/PerpsMarketFactoryModule.sol

```diff
@@ -120,12 +120,15 @@ contract PerpsMarketFactoryModule is IPerpsMarketFactoryModule {
             uint256 collateralValue = globalMarket.totalCollateralValue();
             int256 totalMarketDebt;
 
-            SetUtil.UintSet storage activeMarkets = globalMarket.activeMarkets;
-            uint256 activeMarketsLength = activeMarkets.length();
-            for (uint256 i = 1; i <= activeMarketsLength; i++) {
-                uint128 marketId = activeMarkets.valueAt(i).to128();
-                totalMarketDebt += PerpsMarket.load(marketId).marketDebt(
-                    PerpsPrice.getCurrentPrice(marketId, PerpsPrice.Tolerance.DEFAULT)
+            uint256[] memory activeMarkets = globalMarket.activeMarkets.values();
```

Comment:
this is how the usage of the new oracle manager generally looks in the code: instaed of having a loop over every price which repetedly calls oracle manager, we cal oracle manager once and it processes all the prices at once.